webplugin/js/app/kommunicate.js

```diff
@@ -826,4 +828,23 @@ $applozic.extend(true, Kommunicate, {
         }
         return siblings;
     },
+    btnClickPreChatOpen: function () {
+        var kommunicateIframe = parent.document.getElementById(
```

Comment:
This might break for the pages where CORS policy are applicable
Please check this once

Reply:
It's has been used in multiple times in widget